doc/install.rst

```diff
@@ -1,7 +1,10 @@
-Installation
-============
-CMPLXFOIL is a Python wrapper for the Fortran-based XFOIL code.
-Building and installing the code is a multi-step process that is machine specific.
+Conda Installation
```

Comment:
Also just noticed, could you add
```
************
Installation
************
```
at the top of this page so it appears as a single entry in the left sidebar in the docs?